COBOL Programming Course #2 - Learning COBOL/COBOL Programming Course #2 - Learning COBOL.md

```diff
@@ -0,0 +1,3606 @@
+---
+geometry:
+- margin=1in
+header-includes:
+- \lstset{basicstyle=\ttfamily,
+    stepnumber=2,
+    numbersep=5pt,
+    backgroundcolor=\color{black!10},
+    showspaces=false,
+    showstringspaces=false,
+    showtabs=false,
+    tabsize=2,
+    captionpos=b,
+    breaklines=true,
+    breakatwhitespace=true,
+    breakautoindent=true,
+    linewidth=\textwidth}
+- \hypersetup{colorlinks=true,
+              linkcolor=blue}
+
+
```

Comment:
Can you help add a `\newpage` so that the first chapter is on a new page after the table of content?